collector/filesystem_bsd.go

```diff
@@ -33,6 +33,7 @@ import "C"
 
 const (
 	defIgnoredMountPoints = "^/(dev)($|/)"
+	MNT_RDONLY            = 0x1
```

Comment:
Hm, bit torn here whether to follow go or c naming conventions. Currently the constant gets exported, which doesn't make too much sense. Slightly leaning towards changing that to `mountReadOnly`.